src/components/Dashboard/index.tsx

```diff
@@ -120,95 +145,11 @@ const Index: React.FunctionComponent = () => {
       </PageSection>
       <PageSection>
         <div style={{ marginBottom: '20px' }} />
-        {!isFirstPullDone && (
```

Comment:
With you PR, if you click on Skill or Knowledge and again click on Dashboard, it will show the empty state page and than shows the list of contributions. 

Another scenario, where you are first time logging in, it sometime takes few seconds to fetch the list of PRs, but during that time it shows the empty state, that gives a feeling to the user that they lost their contribution.

To avoid these two scenario, we check for the first pull (isFirstPullDone), if it's not done, it will show a popup that says "Retrieving all your skills and knowledge...".  If there is any better alternative for this with Empty state, i would love to see that, otherwise we should show the popup while the data is loading. We should show empty state, only when there is no existing contribution from user.